src/kernal/screen/e536.init_screen.s

```diff
@@ -0,0 +1,21 @@
+;; #LAYOUT# STD *        #TAKE
+;; #LAYOUT# X16 *        #IGNORE
+;; #LAYOUT# *   KERNAL_0 #TAKE
+;; #LAYOUT# *   *        #IGNORE
+
+;
+; Init screen at reset
+;
+; https://www.c64-wiki.com/wiki/646
+; https://www.lemon64.com/forum/viewtopic.php?t=18490
+;
+; Information on what the code at E536 does is a bit contradictory
+; It seems as if the text color at $286 is set to the value in A
+; and then the screen is cleared.
+; Tests indicate that this is the what happens.
+    
+
+init_screen:
```

Comment:
Please add the information about this routine to https://github.com/MEGA65/open-roms/blob/master/STATUS.md#unofficial-kernal-routineslocations

Most likely this routine does a little bit more (clear screen is at $E544...)

Reply:
I will add that. Yes, it surely does more since there is more room until $E544.

Reply:
I have updated the PR with this